### DISCUSSION TITLE
Don't support MySQL table, you should rebuild Doris with WITH_MYSQL option ON

### DISCUSSION THREAD

**Discussion by BigBig-joe** [apache/doris#23179] [A - General / Q&A] [answered]
Hi all

I got error when I tried to query a external table to Mysql.   the version of doris is 1.2.1. 

I downloaded source package directly [here](https://doris.apache.org/download/), without compiling.
CREATE EXTERNAL TABLE `Airflow_log_1` (
  `id` int(11) NOT NULL,
  `dttm` datetime NULL,
  `dag_id` varchar(250) NULL,
  `task_id` varchar(250) NULL,
  `map_index` int(11) NULL,
  `event` varchar(30) NULL,
  `execution_date` datetime NULL,
  `owner` varchar(500) NULL,
  `extra` text NULL
) ENGINE=MYSQL
COMMENT 'MYSQL'
PROPERTIES (
"host" = "xxxx",
"port" = "3306",
"user" = "airflow",
"password" = "",
"charset" = "utf8",
"database" = "airflow",
"table" = "log"
);

**Comment by zy-kkk** [comment, marked as answer]
![image](https://github.com/apache/doris/assets/70795751/f4bffaf1-ac75-4b5d-a214-4d1762904190)
https://doris.apache.org/docs/dev/install/source-install/compilation-general/#special-statement

**Reply by BigBig-joe** [reply]
is there a higher stable version that can access mysql external table under Apache license 2.0?

**Reply by ixzc** [reply]
You can use 1.2.6

**Reply by BigBig-joe** [reply]
I will try, thanks.

**Reply by BigBig-joe** [reply]
I tried, but unfortunately it still doesn't work.   I think the only solution so far  is compiling with option WITH_MYSQL=1.
